Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Text;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5838869)